students/Ben_freeman/lesson01/test_integration.py

```diff
@@ -0,0 +1,55 @@
+from inventory_management import main
+import io
+import unittest
+from unittest.mock import patch
+
+
+class TestIntegration(unittest.TestCase):
+
+    def test_integration(self):
+        furniture_item = ["1","1", "Chair", "2", "y", "wood", "XL"]
+        electric_item = ["1",'2', "Blender", "3", "n", "y", "samsung", "200v"]
+        other_item = ['1','3', "TheKidFromHomeAlone", "15/hr", "n", "n"]
+        item_code = ["2", "2"]
+        mega_item = furniture_item + electric_item + other_item + item_code
+
+        test_dictionary = {
+            '1': {'productcode': '1',
+                  'description': 'Chair',
+                  'marketprice': 24,
+                  'rentalprice': '2',
+                  'material': 'wood',
+                  'size': 'XL'},
+
+            '2': {'productcode': '2',
+                  'description': 'Blender',
+                  'marketprice': 24,
+                  'rentalprice': '3',
+                  'brand': 'samsung',
+                  'voltage': '200v'},
+
+            '3': {'productcode': '3',
+                  'description':
+                      'TheKidFromHomeAlone',
+                  'marketprice': 24,
+                  'rentalprice': '15/hr'}
+        }
+        item_info_message = "Enter item code: 2\
```

Comment:
This one will work:

`item_info_message = "Please choose from the following options (1, 2, q):\n1. Add a new item to the inventory\n2. Get item information\nq. Quit\nNew inventory item added\nPlease choose from the following options (1, 2, q):\n1. Add a new item to the inventory\n2. Get item information\nq. Quit\nNew inventory item added\nPlease choose from the following options (1, 2, q):\n1. Add a new item to the inventory\n2. Get item information\nq. Quit\nNew inventory item added\nPlease choose from the following options (1, 2, q):\n1. Add a new item to the inventory\n2. Get item information\nq. Quit\nproductcode:2\ndescription:Blender\nmarketprice:24\nrentalprice:3\nbrand:samsung\nvoltage:200v\n"`